validator/client/src/test/java/tech/pegasys/teku/validator/client/duties/AttestationProductionDutyTest.java

```diff
@@ -198,6 +213,11 @@ public void shouldPublishProducedAttestationsWhenSomeUnsignedAttestationsFail()
     verify(validatorLogger)
         .dutyFailed(TYPE, SLOT, Set.of(validator1.getPublicKey().toAbbreviatedString()), failure);
     verifyNoMoreInteractions(validatorLogger);
+
+    verify(validatorDutyMetrics, times(2))
+        .record(any(), any(AttestationProductionDuty.class), eq(Step.CREATE));
+    verify(validatorDutyMetrics, times(2))
+        .record(any(), any(AttestationProductionDuty.class), eq(Step.SIGN));
```

Comment:
This shouldn't happen twice. One of the attestations isn't being produced so we shouldn't count its signing (it never happens).